sig-network/charter.md

```diff
@@ -0,0 +1,46 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
+
+### In scope
+Some notable examples of areas owned by SIG Network:
+- Masquerade, Bridge and Passt binding plugins for the primary network.
+- Bridge, SRIOV and MacVTap bindings for secondary networks.
+- VM networking lifecycle on: creation, migration, hotplug/unplug, restart, stop etc.
+
+#### Code, Binaries and Services
+- [Main repo](https://github.com/kubevirt/kubevirt).
+- Networking related APIs on: VM, VMI, Kubevirt CR, etc.
+- Core network binding infrastructure.
+- Network binding plugin infrastructure.
+- Examples of binding plugin CNIs and sidecars.
+- Virtual Machine/Virtual Machine Instance networking status management.
+- Domain networking configuration.
+- Pod network definition.
```

Comment:
This one is pretty vogue, does it mean connecting the guest to pod network?

Reply:
yes

Reply:
@ormergi Can I resolve this comment or do you want something to be changed here?

Reply:
No blocker from my side, it a nit and I already approved this PR on previous review cycle :slightly_smiling_face: 
https://github.com/kubevirt/community/pull/271#pullrequestreview-1970215570